### ISSUE TITLE
Is it dead ?

### ISSUE THREAD

**Issue by aviallon** [ircmaxell/php-compiler#79]
The last commit is from 3 years ago...
is the project dead ?

**Comment by PurHur** [comment]
I guess you could create a new PR to revive it!

**Comment by aviallon** [comment]
> I guess you could create a new PR to revive it!

Indeed. Perhaps another day, when I have more free time on my hands.

**Comment by pmaasz** [comment]
after 2,5 years... congratulations on becoming a dad, @aviallon.

**Comment by zaunere** [comment]
"Now, thanks to [FFI landing in PHP 7.4](https://wiki.php.net/rfc/ffi), the potential for all sorts of crazy is HUGE."
Looking forward to the all sorts of crazy:  PHP -> LLVM/MLIR => many many sorts of crazy ??

**Comment by PurHur** [comment]
For you really interested guys.I throwed multiple metric tons of ai onto this repo.
https://github.com/PurHur/php-compiler
